src/pyk/kast/outer_lexer.py

```diff
@@ -57,56 +58,74 @@ class TokenType(Enum):
     BUBBLE = auto()
 
 
+class Loc(NamedTuple):
+    line: int
```

Comment:
```suggestion
    row: int
```

(This is a nitpick, so feel free to disregard the suggestion.)